railties/test/generators/app_generator_test.rb

```diff
@@ -746,17 +746,38 @@ def test_spring_with_dev_option
   def test_webpack_option
     command_check = -> command, *_ do
       @called ||= 0
-      @called += 1 if command == "webpacker:install"
-      assert_equal 1, @called, "webpacker:install expected to be called once, but was called #{@called} times."
+      if command == "webpacker:install"
+        @called += 1
+        assert_equal 1, @called, "webpacker:install expected to be called once, but was called #{@called} times."
+      end
```

Comment:
I noticed that this test wasn’t 100% correct. We shouldn't assert the `@called` if `command` isn't “webpacker:install". (e.g. "active_storage:install’)